notebooks/Datasets_1_Prepare_PubChem_Datasets.ipynb

```diff
@@ -0,0 +1,1059 @@
+{
```

Comment:
very nice!  Lots of cool tricks in here, I'm excited to give it a more thorough look soon.